nodes/vector/turbulence.py

```diff
@@ -124,21 +123,12 @@ def process(self):
         m_freq = inputs['Frequency'].sv_get()[0]
         m_seed = inputs['Random seed'].sv_get()[0]
 
-        #if verts and verts[0]:
-
         maxlen = len(verts[0])
```

Comment:
you can also move the function definitions for `get_offset` and `seed_adjusted` outside of the node, keep the process function shorter.